webapp/package.json

```diff
@@ -10,11 +10,11 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "cssmin": "0.4.3",
-    "eslint": "7.13.0",
-    "eslint-config-prettier": "6.15.0",
-    "eslint-plugin-prettier": "3.1.4",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^10.0.1",
+    "eslint-plugin-prettier": "^5.2.3",
     "graceful-fs": "4.2.4",
-    "prettier": "2.1.2",
+    "prettier": "^3.5.0",
```

Comment:
Why define a minimum rather than pinning to a specific version?

Reply:
@willkg when it's not a critical dependency I often use a minimum in case of minor patches.  But I'm happy to pin it if you prefer!

Reply:
We update this stuff rarely, so I'm wondering whether it's better to pin it and then re-enable dependabot to update things. That way it doesn't change without us approving it.